betelgeuse.py

```diff
@@ -209,13 +223,29 @@ def test_case(path, collect_only, project):
                     test_case.update()
 
 
-@cli.command('test-run')
+@cli.command('test-results')
 @click.option(
     '--path',
     default='junit-results.xml',
     help='Path to the jUnit XML file.',
     type=click.Path(exists=True, dir_okay=False),
 )
+def test_results(path):
```

Comment:
@sthirugn sorry, I made a rookie mistake and suffered from 'copy and paste' syndrome :/ I've updated the docstring, let me know what you think